HousingRepairsOnlineApi/UseCases/RetrieveAvailableAppointmentsUseCase.cs

```diff
@@ -18,16 +20,32 @@ public RetrieveAvailableAppointmentsUseCase(IAppointmentsGateway appointmentsGat
             this.sorEngine = sorEngine;
         }
 
-        public async Task<IEnumerable<Appointment>> Execute(string repairLocation, string repairProblem, string repairIssue, string uprn)
+        public async Task<List<ApplicationTime>> Execute(string repairLocation, string repairProblem,
+            string repairIssue, string uprn)
```

Comment:
Please rename `uprn` to `locationId` to avoid confusion about the value of the field.